### PR TITLE
Update k8s to v1.7.2, heapster, helm, registry, dashboard, etcd

### PR DESCRIPTION
**Pull request by dkoshkin**
As part of this, removing the `"Installing with previous version of Kismatic"` test. This was initially added after the package repo was broken, however the `upgrade` tests are also testing this.
It is also cumbersome to maintain by manually changing the previous version.

### CODE REVIEW

File: integration/plan_patterns.go [hunk 1]
```diff
@@ -44,6 +44,7 @@ const planAWSOverlay = `cluster:
   disconnected_installation: {{.DisconnectedInstallation}}
   disable_registry_seeding: {{.DisableRegistrySeeding}}
   networking:
+    type: overlay
```
Comment (alexbrand) [inline integration/plan_patterns.go]: can we add a comment on these that they are here because of backwards compat.

**Comment by alexbrand** [comment]
LGTM

**Comment by swade1987** [comment]
@alexbrand @dkoshkin are we going to be using Helm v2.5.1 in the v1.5.0 release of KET?

**Comment by alexbrand** [comment]
@swade1987 Yep! Helm v2.5.1

**Pull request merged**